templates/pages/cart.html

```diff
@@ -4,7 +4,7 @@
 {{inject 'cancelButtonText' (lang 'common.cancel')}}
 {{#partial "page"}}
 {{inject 'invalidEntryMessage' (lang 'cart.invalid_entry_message')}}
-<div class="page">
+<main class="page">
 
     <div class="page-content" data-cart>
         {{> components/common/breadcrumbs breadcrumbs=breadcrumbs}}
```

Comment:
to discuss:
Should the repeating `breadcrumbs` component be inside the `<main>` tag, which should contain unique content?
And is the `breadcrumbs` component considered as unique, since it may be partially different on each page, for example:
```
Home > All products > T-shirt
Home > All products > Brooklyn Bicycle
```